content/docs/1.7.0/snapshots-and-backups/backup-and-restore/create-a-backup.md

```diff
@@ -16,4 +16,29 @@ To create a backup,
 
 **Result:** The backup is created. To see it, click **Backup** in the top navigation bar.
 
-For information on how to restore a volume from a snapshot, refer to [this page.](../restore-from-a-backup)
\ No newline at end of file
+For information on how to restore a volume from a snapshot, refer to [this page.](../restore-from-a-backup)
+
+## Full Backup
+
+By default, Longhorn performs delta backup on the snapshot, meaning it only backs up data that has been newly updated since the last backup. This approach enhances time efficiency and conserves network throughput. However, in the event of a corrupted data block in the backupstore, Longhorn does not replace it with a healthy one during subsequent backup operations.
+
+Starting with v1.7.0, Longhorn allows users to perform full backup to upload all the data blocks of the volume. Longhorn overrides data block on the backup store if it already exists.
+
+To create a backup,
```

Comment:
In addition to UI, can we also provide a headless way too to create a full backup by kubectl? 

P.S. for any operations, it's better to have both ways described. UI and CLI.

Reply:
added